client/src/components/Menu.tsx

```diff
@@ -21,27 +21,27 @@ const links = [
 ]
 
 export const Menu = () => {
-    const location = useLocation();
-    const [activeLink, setActiveLink] = useState('');
-    const navigate = useNavigate();
-    const [currentPage, setCurrentPage] = useState('');
-    const isLinkAnchor = (linkTo: string) => linkTo === '/' || linkTo.startsWith('/#');
-    
+    const location = useLocation()
+    const [activeLink, setActiveLink] = useState('')
+    const navigate = useNavigate()
+    const [currentPage, setCurrentPage] = useState('')
+    const isLinkAnchor = (linkTo: string) => linkTo === '/' || linkTo.startsWith('/#')
+
```

Comment:
Looks like there's a setting mismatch in your prettier, not a big issue for now, we can touch base on that next